src/server/helpers/utils.js

```diff
@@ -226,14 +226,15 @@ export function truncateTables(Bookshelf: Object, tables: Array<string>) {
  * Return additional relations to withRelated array according to modelType
  *
  * @param {string} modelType - type of the model or entity
- * @returns {array} array of addition relations
+ * @returns {array} array of additional relations
  */
 export function getAdditionalRelations(modelType) {
+	const additionalRelations = [];
 	if (modelType === 'Work') {
-		return ['disambiguation', 'workType'];
+		return additionalRelations.concat(['disambiguation', 'workType']);
```

Comment:
Sorry, I wasn't very clear with this. 
I didn't mean to say to use concat here, but rather to always return an array, so that in another file I can call `myArray.concat(getAdditionalRelations(...))`.

So you can return array as you did before, but change the last line to be `return []`.